timemachine/fe/single_topology.py

```diff
@@ -1053,28 +1055,15 @@ def _setup_intermediate_chiral_bond_term(self, src_bond, dst_bond, lamb, interpo
 
         return potentials.ChiralBondRestraint(chiral_bond_idxs, chiral_bond_signs).bind(chiral_bond_params)
 
-    def setup_intermediate_state(self, lamb, lambda_angles=0.4, lambda_torsions=0.7):
-        """
+    def setup_intermediate_state(self, lamb):
+        r"""
         Setup intermediate states at some value of lambda.
 
         Parameters
         ----------
         lamb: float
-
-        lambda_angles, lambda_torsions: float
```

Comment:
Some of the information in this comment seems like useful context, perhaps keeping the bits about how the interpolation works generally would be good.

Reply:
Restored and expanded in https://github.com/proteneer/timemachine/pull/938/commits/8939e562b10f590e032e03be38371f2794fe7db4